components/button-group/index.jsx

```diff
@@ -102,6 +99,15 @@ const ButtonGroup = (props) => {
 			</fieldset>
 		);
 	}
+
+	if (props.variant === 'list') {
+		return (
+			<ul className={classNames('slds-button-group-list', props.className)}>
+				{children}
```

Comment:
Since every child within ButtonGroup of variant `list` is meant to be wrapped in an `li` tag, please make this code do that here. Something like this:
```
{React.Children.map(props.children, (child) => (
	<li>{child}</li>
))}
```

Reply:
Done.